### PR TITLE
Refactors how magazine ammo color is handled into overlays

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Refactors how magazine ammo color is handled into overlays.

Instead of filling up dmis with a ridiculous amount of icon states for each new barely used magazine type, compatible magazines have a 'band' white overlay icon that is colored based on a variable on the magazine.

This will cause various sprites of various magazines to look subtly different as the exact look couldn't be copied.

Renamed wallpiercing to wall-penetrating.

Removed cluster magazines from the code.

Altered the name of A19 magazines a little.

# Explain why it's good for the game

> Refactors how magazine ammo color is handled into overlays.
> Instead of filling up dmis with a ridiculous amount of icon states for each new barely used magazine type, compatible magazines have a 'band' white overlay icon that is colored based on a variable on the magazine.
> This will cause various sprites of various magazines to look subtly different as the exact look couldn't be copied.

This will help a lot if adding new magazines as you don't have to sift through the awful, bloated dmis. Additionally, it's been proven that more icons in a dmi causes lag, so the less the merrier.

> Renamed wallpiercing to wall-penetrating.

More accurate

> Removed cluster magazines from the code.

These didn't fit with the new icon handling system, are not used anywhere, and aren't interesting enough to be worth staying in the code.

> Altered the name of A19 magazines a little.

So i can do 'HV high impact'

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:

refactor: Refactors how magazine ammo color is handled into overlays.
refactor: Instead of filling up dmis with a ridiculous amount of icon states for each new barely used magazine type, compatible magazines have a 'band' white overlay icon that is colored based on a variable on the magazine.
imageadd: This will cause various sprites of various magazines to look subtly different as the exact look couldn't be copied.
spellcheck: Renamed wallpiercing to wall-penetrating.
code: Removed cluster magazines from the code.
spellcheck: Altered the name of A19 magazines a little.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry_reagents/medical.dm [hunk 1]
```diff
@@ -269,7 +269,7 @@
 	properties = list(PROPERTY_NEOGENETIC = 3)
 
 /datum/reagent/medical/quickclot
-	name = "Quick Clot"
+	name = "wall-penetrating"
```
Comment (Segrain) [inline code/modules/reagents/chemistry_reagents/medical.dm:272]:
```suggestion
	name = "Quick Clot"
```
I do not even know _how_.

Comment (carlarctg) [reply code/modules/reagents/chemistry_reagents/medical.dm:272]: image

File: html/changelogs/AutoChangeLog-pr-2762.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+author: "realforest2001"
+delete-after: True
+changes:
+  - rscadd: "Re-Adds autoreplies to ahelps."
```
[the remark this answers is not in the document]
Comment (carlarctg) [reply html/changelogs/AutoChangeLog-pr-2762.yml:4]: the uck

**Comment by cm13-github** [comment]
#### Test Merge Deployed

<details><summary>Details</summary>

##### Server Instance
CM13 Live Instance

##### Revision
Origin: 4708853a780fe8dc72848aaef4a440011ac4490f
Pull Request: 24b3a279ac07f83dfab648500d91b868c1973f23
Server: 2553f9eae54e1e8fba39250f1fbaef3aad25d1af

##### Comment

</details>

**Comment by cm13-github** [comment]
#### Test Merge Updated

<details><summary>Details</summary>

##### Server Instance
CM13 Live Instance

##### Revision
Origin: 4708853a780fe8dc72848aaef4a440011ac4490f
Pull Request: 24b3a279ac07f83dfab648500d91b868c1973f23
Server: cdd6fe2764fa15eae4632c12aeee0e9d5ece932e

##### Comment

</details>

**Review by harryob — APPROVED** [review]

**Comment by cm13-github** [comment]
#### Test Merge Removed

**Pull request merged**